src/Commands/RemindCommandGroup.cs

```diff
@@ -163,10 +157,22 @@ private async Task<Result> ListRemindersAsync(MemberData data, IUser executor, I
                 string.Format(Messages.ReminderCreated, executor.GetTag()), executor)
             .WithDescription(builder.ToString())
             .WithColour(ColorsList.Green)
-            .WithFooter(string.Format(Messages.ReminderPosition, memberData.Reminders.Count))
+            .WithFooter(string.Format(Messages.ReminderPosition, memberData.Reminders.Count + 1))
             .Build();
+        var messageResult = await _feedback.SendContextualEmbedAsync(embed.Entity, ct: ct);
+        if (!messageResult.IsDefined(out var output)) { return (Result)messageResult; }
```

Comment:
As I remember, we don't use one-line `if`s
```suggestion
        if (!messageResult.IsDefined(out var output))
        {
            return (Result)messageResult;
        }
```